tests/expressions_case/tests.py

```diff
@@ -21,6 +22,7 @@
     Image = None
 
 
+@ignore_warnings(category=RemovedInDjango20Warning)
```

Comment:
Have you read https://docs.djangoproject.com/en/1.9/internals/contributing/writing-code/submitting-patches/#deprecating-a-feature about how this works? Since these deprecations use the system check framework, there will only be a problem if it's invoked against one of the test models.